include/js/general.js

```diff
@@ -7633,6 +7633,33 @@ function openWizard(mapid) {
 	}, 100);
 }
 
+async function handleDrawClick(checkmodule, checkaction, recordid, doc2edit = '') {
+	const response = await ExecuteFunctions('ispermitted', `checkmodule=${checkmodule}&checkaction=${checkaction}&checkrecord=${recordid}`);
+
+	if (!response) {
+		try {
+		  ldsModal.close(true);
+		} catch (error) {}
```

Comment:
I think our formatting rules indicate that the keys have to be in separate lines